packages/ag-charts-website/src/content/docs/error-bars/_examples/scatter-series-error-bars/main.ts

```diff
@@ -0,0 +1,25 @@
+import { AgChartOptions, AgEnterpriseCharts } from 'ag-charts-enterprise';
+import { getData } from './data';
+
+const options: AgChartOptions = {
+    container: document.getElementById('myChart'),
+    data: getData(),
+    title: {
+        text: 'Volume-Pressure Relationship with Confidence Intervals',
+    },
+    series: [
```

Comment:
Note: This scatter plot doesn't show units for X (m³) and Y (Bar) axes.

Also this generated data is counter-intuitive from a physics point of view. It shows a positive correlation between Volume and Pressure, but this should be reversed ([Boyle's law](https://en.wikipedia.org/wiki/Boyle%27s_law#:~:text=Boyle%27s%20law%20is%20a%20gas,doubled%2C%20the%20pressure%20is%20halved.)).

We may want to consider tweaking this example chart & data for better presentation.

Reply:
I've reverse the relationship between Volume & Pressure, but the units of the axes still isn't show. Please let me know whether that's acceptable for this example.

Reply:
So there are a few relevant configuration points depending on how you want to display the units:
- `series.label.formatter` - series label format
- `series.tooltip.format` - tooltip format
- `axes[].label.format` - axis label format
- `axes[].label.formatter` - axis label formatter
- `axes[].title.text` - axis title

For examples I would avoid adding too much clutter, so whichever gives a sensible result in the smallest number of additional lines is probably best.

Reply:
Understood. I think that we can leave it like this for the documentation and leave the more stylised examples for the showcase.